core/domain/story_domain.py

```diff
@@ -665,8 +665,8 @@ def get_acquired_skill_ids_for_node_ids(self, node_ids):
             the story.
 
         Returns:
-            list(str)|None. The list of acquired skill ids for the
-                node ids or None, if no node is linked to them.
+            list(str). The list of acquired skill ids for the node ids.
```

Comment:
"The union of the acquired skill IDs corresponding to each of the node IDs."

(And then, the second sentence is optional.)

Reply:
Done.